src/com/ms/silverking/cloud/dht/daemon/storage/NamespaceStore.java

```diff
@@ -982,11 +982,11 @@ protected final OpResult _put(DHTKey key, ByteBuffer value, StorageParametersAnd
         }
         
         if (debugVersion) {
```

Comment:
debugVersion is off by default. Make Log.warningAsyncf so that the debugVersion flag has an effect